src/Components/Favorites.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import '../Styles/Favorites.css'
+import { useDispatch, useSelector } from 'react-redux'
+import { NavLink } from 'react-router-dom'
+import { removeFavoriteDrink } from '../redux/actions/favoriteDrinkAction'
+
+
+const Favorites = () => {
+    const favoriteDrinks = useSelector(state => state.favoriteDrinks.favoriteDrinks)
+    const dispatch = useDispatch()
+
+    const drinks = favoriteDrinks.map(drink => {
+        if (favoriteDrinks) {
+            return (
+                <div className='fav-drink-card' data-cy='favorite-card'>
+                    <div className='left-side-card'>
+                        <h3 className='fav-drink-name'>{drink.strDrink}</h3>
+                        <button className='remove-button' data-cy='remove-button' onClick={() => dispatch(removeFavoriteDrink(drink.idDrink))}>Remove</button>
+                    </div>
+                        <img className='fav-drink-image' src={drink.strDrinkThumb}/>    
+                </div>
+            )
+        } 
+    })
+
+    
+    return (
+        <div>
+            <NavLink to={'/'} style={{textDecoration: 'none'}}>
```

Comment:
Should the NavLink be a Link instead? My understanding of the NavLink is that you use it when you have several links together and they remain visible after navigation and the active link is highlighted. What do y'all think?

Reply:
Oh - good catch @raz-joan  - I think this can be just regular ole Link component